src/main/java/com/algosenpai/app/response/README.md

```diff
@@ -0,0 +1,3 @@
+### Response
```

Comment:
Isn't this part of UI also?

Reply:
I will remove request and response package

Reply:
removing it